docs/self-managed/reference-architecture/kubernetes.md

```diff
@@ -0,0 +1,287 @@
+
+### Kubernetes
+
+:::note
+The reference architecture overview describes the difference between the [Orchestration cluster and Web Modeler and Console](/docs/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console), and provides additional information on application communication.
+:::
+
+A [multi-namespace deployment](/self-managed/setup/guides/multi-namespace-deployment.md) is recommended. For more information, see the [components](#components) section.
+
+The following depictions provide a simplified view of the deployed namespaces using the [Camunda 8 Helm chart](/self-managed/setup/install.md). To keep the diagram clear, ConfigMaps, Secrets, RBAC, and ReplicaSets have been omitted.
+
+#### Orchestration cluster
+
+![Orchestration Cluster](./img/k8s-cluster-view-orchestration.jpg)
+
+By default, the Helm chart suggests using a single Ingress for Camunda resources, enabling a unified domain with each application accessible via a different path.
+
+Most applications are stateless and deployed as **Deployments**. However, Zeebe Brokers are an exception, requiring a **StatefulSet** to ensure that volumes are consistently mounted, as pod order and identifiers are crucial.
+
+Zeebe Brokers also have a service but are not directly exposed externally; all requests must pass through the Zeebe Gateway, which is typically used for internal communication as well.
+
+#### Web Modeler and Console
+
+![Web Modeler and Console](./img/k8s-cluster-view-managing.jpg)
+
+Web Modeler, Console, and Identity are deployed as **Deployments** since they are stateless, with data stored externally in a SQL database. This allows them to be easily scaled as needed.
+
+The namespace has its own Ingress resource, as those are namespace-bound and not cluster-wide resources. This means you will have to use a different subdomain for each Ingress resource. For more details, refer to the [multi-namespace deployment guide](/self-managed/setup/guides/multi-namespace-deployment.md).
+
+### High availability (HA)
+
+:::caution Non-HA importer / archiver
+The following concerns **Operate**, **Tasklist**, and **Optimize**:
+
+When scaling from a single pod to multiple, ensure that the `importer / archiver` is enabled on only one pod and disabled on others. Enabling it on multiple pods will cause data inconsistencies. This limitation is known and will be addressed in future updates.
+
+Example configuration options are available for [Operate](/self-managed/operate-deployment/importer-and-archiver.md#configuration) and [Tasklist](/self-managed/tasklist-deployment/importer-and-archiver.md#configuration).
```

Comment:
I couldn't find the configuration examples for Optimize. 😬 Not sure if that should be flagged elsewhere to add in the docs, or if the configuration is identical. We could just mention that if so.